tools/preconf-rpc/main.go

```diff
@@ -203,6 +203,13 @@ var (
 		EnvVars: []string{"PRECONF_RPC_WEBHOOK_URLS"},
 	}
 
+	optionAuthToken = &cli.StringFlag{
```

Comment:
Do we need a refresh interval of 1h for the auth token if it's only loaded on startup? Is the token intended to rotate every hour?